wordpress-mobile/WordPress-iOS
# Fix iOS 11 asset catalog color loading bug.

This PR adds a temporary workaround to fix a bug with the Xcode 11 GM seed where colors can't be loaded from an asset catalog in iOS 11. I generated a struct to load colors and am using that on iOS < 12. Apple have stated that they should fix the bug in a later GM seed, so hopefully we can remove this fix soon, but we'll want to re-test it later.

**To test:**

* In Xcode 11, install an iOS 11 simulator, or use an iOS 11 device.
* Build and run on iOS 11, and run through some flows to check the colors in the app look correct. If you build and run without this patch, all colors that would be loaded from the asset catalog appear as red.
* Double check things still look ok on iOS 12 or 13.

Update release notes:
